frame/democracy/src/lib.rs

```diff
@@ -367,6 +393,11 @@ decl_storage! {
 
 		/// Record of all proposals that have been subject to emergency cancellation.
 		pub Cancellations: map hasher(identity) T::Hash => bool;
+
+		/// Storage version of the pallet.
+		///
+		/// New networks start with last version.
+		StorageVersion build(|_| Some(Releases::V1)): Option<Releases>;
```

Comment:
I want to note that there's a slightly inconsistency compared with how other modules define `StorageVersion` -- they're usually defined without `Option` and default to `V1`, for example, in `balances` pallet. https://github.com/paritytech/substrate/blob/master/frame/balances/src/lib.rs#L380

Reply:
Yeah, `Balances` will break when someone forgets to update the linked line for new chains. This is clearly we should prevent.

Reply:
@bkchr TBH I think it's the same -- this is defined in the pallet itself, not directly accessible by users, so I don't think either case will allow any misuse. Both are fine, but just slightly different/inconsistent.

Reply:
all in all it doesn't change much, also not having to implement a missnamed `Default` feels to me better

Reply:
Ahh fuck, now I see what you mean. 

But in overall you are right that it is not good, but I'm not happy with all the strategies at the moment.